CHANGES.md

```diff
@@ -2,6 +2,13 @@
 
 ## not yet released
 
+## 2.3.1
+
+- `MANTA-4799 add branch protections on "mantav1" branches of the relevant manta repos`
+  This updates the `jr gh` `check` and `set-branch-protection` subcommands so
+  that they also operate on the `mantav1` branch of the given repository or
```

Comment:
```suggestion
  that they also operate on the `mantav1` branch protection settings of the given repository or
```

Reply:
^^ to not scare me that that command changes my actual repo branch! :)

Reply:
good point, I'll fix that.